### DISCUSSION TITLE
Change language with accents

### DISCUSSION THREAD

**Discussion by eaguirres** [webfashionist/RichText#112] [Q&A] [answered]
Hi, I want to set the language to Spanish. I guess I have to manually change each property within translations. The problem is that the accents are not displayed correctly, how can I display the accents?

**Comment by webfashionist** [comment]
Hi @eaguirres 

Thank you for your question. 

What do you mean, that they are not displayed correctly? Have you checked if the charset of the page/HTML file is UTF-8?

You can try using 
```html
<meta charset="UTF-8">
```
in the `<head>` of your page to try it out.

**Comment by eaguirres** [comment, marked as answer]
I got it to work. The jquery.richtext.js had an ANSI encoding, I changed it to UFT-8 and it worked. Although I used <meta charset="UTF-8"> my site did not show the accents. 

Thanks!
